### DISCUSSION TITLE
Collecting Requirements/Ideas/Features for v1.11.0

### DISCUSSION THREAD

**Discussion by OrlinVasilev** [vmware-tanzu/velero#5467] [General discussions]
Hello Community!

Velero v1.10 is coming within the next weeks, and before we start working on v1.11, we’d love to brainstorm with you on ideas and requirements for the feature list of this next release!

We will be super grateful if you submit your ideas and feature request along side with your willingness to work on them, which will help tremendously the project and community!

Please review what we have already laid out in [the v1.11 roadmap](https://github.com/vmware-tanzu/velero/wiki/1.11-Roadmap) and share your thoughts/ideas/requirements/feedback here so we can all work together on building out the next version of Velero.

Orlix
Velero Community Manager

**Comment by weshayutin** [comment]
@OrlinVasilev Thank you so much for starting this thread.  It's very helpful to be able to plan for an upcoming release of velero.  

**Designs:** 
1.  [Item action progress monitoring design](https://github.com/vmware-tanzu/velero/pull/5108/)
2. [BackupItemAction v2 design](https://github.com/vmware-tanzu/velero/pull/5382)
3. [RestoreItemAction v2 design]( https://github.com/vmware-tanzu/velero/pull/5408)
4. [VolumeSnapshotter v2 API design](https://github.com/vmware-tanzu/velero/pull/5421)

**GH Issues related to Plugin v2 API changes:**
- [Implement new plugin v2 APIs for VolumeSnapshotter](https://github.com/vmware-tanzu/velero/issues/5487 )
- [Implement new plugin v2 APIs for BackupItemAction](https://github.com/vmware-tanzu/velero/issues/5488)
- [Implement new plugin v2 APIs for RestoreItemAction](https://github.com/vmware-tanzu/velero/issues/5489)

**GH Issues related to the implementation of BR monitoring:**

- [Implement backup monitoring for backupItemAction](https://github.com/vmware-tanzu/velero/issues/5490)
- [Implement restore monitoring for restoreItemAction](https://github.com/vmware-tanzu/velero/issues/5497)

**GH Issues related to B/R phases:**

- [New Backup phases:]( https://github.com/vmware-tanzu/velero/issues/5491 )
- [New Restore phases](https://github.com/vmware-tanzu/velero/issues/5492)
- [Defer uploading backup.json](https://github.com/vmware-tanzu/velero/issues/5493 )
- [Implement <backup-name>-itemoperations.json.gz file](https://github.com/vmware-tanzu/velero/issues/5494 )
- [Implement <restore-name>-itemoperations.json.gz file restart logic](https://github.com/vmware-tanzu/velero/issues/5495 )
- [Change recon logic to ignore B/R that are not completed](https://github.com/vmware-tanzu/velero/issues/5496 )

**GH Issue (to be created ) describing holistic changes for timeouts:**

- Discover timeouts through out the velero codebase
  - Understand the existing related parameters
  - Identify hard coded timeouts
  - Propose a logical mapping for remaining hardcoded timeouts
